cucumber_task/features/task.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Website testing
+
+Scenario: Patient table is not empty
+  Given patient page is opened
+  When opening patient page
+  Then patient table is not empty
+
+Scenario: User can register a new patient
+  Given patient page is opened
+  When user fills all fields on patient register page
+  Then new patient is created
+
+Scenario: User can not register an existing patient
+  Given patient page
+  When register patient is opened and all data fields are filled
+  Then user can not create an existing patient
+
+Scenario: User can add a new task in the system administration page
+  Given system administration page is opened
+  When user click on the add task page and fill all fields by using valid data
+  Then user can see added tasks on the page
+
+Scenario: User can merge records of two patients
+  Given data management page is opened
+  When user click on the merge patient records
```

Comment:
how the system should defer which step definition to use, you have 3 versions of the same step?